specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -2680,10 +2689,10 @@
         "TopologyResourceName": {
             "name": "topologyResourceName",
             "in": "path",
-            "description": "Name of an topology resource group.",
+            "description": "Name of a topology resource group.",

```

Comment:
[continuation of previous discussion]
the term "resource group" is already in use so it might confuse customers, please change the description